--- a/qtwebengine/src/pdf/CMakeLists.txt
+++ b/qtwebengine/src/pdf/CMakeLists.txt
@@ -52,10 +52,6 @@
 #  PDF DOCS
 ##
 
-qt_internal_add_docs(Pdf
-    doc/qtpdf.qdocconf
-)
-
 add_code_attributions_target(
     TARGET generate_pdf_attributions
     OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/pdf_attributions.qdoc
@@ -63,7 +59,6 @@
     BUILDDIR ${buildDir}/${config}/${arch}
 )
 add_dependencies(generate_pdf_attributions run_pdf_GnDone)
-add_dependencies(prepare_docs_Pdf generate_pdf_attributions)
 
 ##
 #   TOOLCHAIN SETUP
